### ISSUE TITLE
Picture should have alt text so that copying does something useful

### ISSUE THREAD

**Issue by jsoref** [github/docs#42697]
### Code of Conduct

- [x] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/account-and-profile/how-tos/organization-membership/publicizing-or-hiding-organization-membership

(And every other article that uses pictures to describe things.)

### What changes are you suggesting?

Add alt text.

### Additional information

I was trying to copy the instructions from https://docs.github.com/en/account-and-profile/how-tos/organization-membership/publicizing-or-hiding-organization-membership into an email:

```
In the upper-right corner of GitHub, click your profile picture, then click  Organizations.

Click the name of your organization.

Under your organization name, click  People.

Screenshot of the horizontal navigation bar for an organization. A tab, labeled with a person icon and "People," is outlined in dark orange.
Locate your username in the list of members. If the list is large, you can search for your username in the search box.

Next to your username, select the visibility dropdown menu, then click a new visibility.

To publicize your membership, choose Public.
To hide your membership, choose Private.
Screenshot of an entry in the list of organization members. Next to the username, a dropdown menu, labeled "Private", is outlined in dark orange.
```

What I got (in each instance) was a doubled space:
```
In the upper-right corner of GitHub, click your profile picture, then click__Organizations.
...
Under your organization name, click__People.
```

It's useful for people reading the text I copied to know that the instructions were actually talking about an icon. But instead, it just looks like I wrote two spaces for no particular reason.

**Comment by Sharra-writes** [comment]
@jsoref I know having some kind of description for images in general is an accessibility issue that we would need to correct asap if we don't have anything for icons, but I suspect there's some other solution for icons that we're using. Still, I'm looking into it to see what we can do and to make sure the accessibility is there.

**Comment by yanskie24143-tech** [comment]
kayi08

**Comment by indragunawan98** [comment]
Duplicate of #43309

**Comment by subatoi** [comment]
@jsoref I agree with you, and whilst the docs are subject to regular accessibility audits, there's still room for improvement with respect to images in docs. I can't guarantee a timeframe for action amongst other priorities but given the scope of this (it's not one article), I hope you'll agree that the most appropriate way to track this is internally—it is, indeed, already tracked there. I'm going to close this out, but thank you for raising it 🙇

**Comment by jsoref** [comment]
- filed https://github.com/community/community/discussions/206710
